ote/src/cljs/ote/ui/buttons.cljs

```diff
@@ -51,3 +51,17 @@
              (stylefy/use-style style-buttons/svg-button)
              opts)
    label])
+
+(defn big-icon-button-with-label
+  "Give icon-element in format: (ic/action-description {:style {:width 30
+                                                        :height 30
+                                                        :margin-right \"0.5rem\"
+                                                        :color colors/primary}})
+  And label-text as text.
+  element-tag is :button or :a depending on do you want to move the user to different page or do magic right where she/he is."
+  [options icon-element label-text element-tag]
+  [element-tag (merge options
+                  (stylefy/use-style style-base/blue-link-with-icon)
+                  (stylefy/use-style style-buttons/outline-button))
+   icon-element
+   [:span {:style {:padding-top "7px"}} label-text]])
```

Comment:
Isn't the sort-of policy nowadays to have relative units for styles unless there's an explicit reason to have px?

Reply:
Yes. but how do you center texts when they are depending on icon sizes? This way I could make this work.

Reply:
If both icon and text are ok to be on same x-axis, then parent: `display: flex`, `align-items: center`
refer to e.g. warning-msg which uses `msg-container`

Reply:
fixed